src/main/java/cuchaz/enigma/utils/LangUtils.java

```diff
@@ -0,0 +1,74 @@
+package cuchaz.enigma.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Map;
+
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+
+import cuchaz.enigma.config.Config;
+
+public class LangUtils {
+	public static final String DEFAULT_LANGUAGE = "en_us";
+	private static final Gson GSON = new Gson();
+	
+	public static String translate(String key) {
+		return translate(key, Config.getInstance().language);
+	}
+	
+	public static String translate(String key, String language) {
+		Path path = Paths.get("src/main/resources/lang/" + language + ".json");
+		Path defaultPath = Paths.get("src/main/resources/lang/" + DEFAULT_LANGUAGE + ".json");
```

Comment:
Also at runtime you don't get from `src/main/resources`; you get from `ClassLoader.findResource` etc. etc.